contrib/langchain/README.md

```diff
@@ -1,12 +1,12 @@
 <h3 align="center">
   <a name="readme-top"></a>
   <img
-    src="https://docs.arcade-ai.com/images/logo/arcade-ai-logo.png"
+    src="https://docs.arcade.dev/images/logo/arcade-logo.png"
   >
 </h3>
 <div align="center">
-  <h3>LangChain Integration</h3>
-    <a href="https://github.com/arcadeai/arcade-ai/blob/main/LICENSE">
+  <h3>Arcade Langchain Integration</h3>
+    <a href="https://github.com/arcadeai/langchain-arcade/blob/main/LICENSE">
```

Comment:
should it be `https://github.com/ArcadeAI/arcade-ai/blob/main/contrib/langchain/LICENSE` instead?